src/item/item_note.cpp

```diff
@@ -234,7 +234,7 @@ void ItemNote::contextMenuEvent(QGraphicsSceneContextMenuEvent* event)
     ungrabMouse(); //Important line! Otherwise the element will be moved after you close the context menu and draw a selection rectangle somewhere.
 
     if (sel_action == act_remove) {
-        remove();
+        delete this;
```

Comment:
Same here, let's set `this = nullptr` afterwards to prevent nasty runtime errors.